src/pipeline.ts

```diff
@@ -793,6 +793,7 @@ export class GitHubWorkflow extends PipelineBase {
         with: {
           name: output.fileSet.id,
           path: output.directory,
+          ['include-hidden-files']: true,
```

Comment:
This I would check to see what we are actually including in the artifact and why.